test/metabase/lib/filter_test.cljc

```diff
@@ -1,30 +1,83 @@
 (ns metabase.lib.filter-test
   (:require
-   [clojure.test :refer [deftest is testing]]
+   [clojure.test :refer [deftest is]]
+   [clojure.test.check.generators :as gen]
+   [clojure.walk :as walk]
+   [com.gfredericks.test.chuck.clojure-test :as chuck.test :refer [checking]]
+   [malli.core :as mc]
+   [malli.generator :as mg]
    [metabase.lib.core :as lib]
    [metabase.lib.metadata :as lib.metadata]
-   [metabase.lib.test-metadata :as meta])
-  #?(:cljs (:require [metabase.test-runner.assert-exprs.approximately-equal])))
+   [metabase.lib.schema.filter :as schema.filter]
+   [metabase.lib.test-metadata :as meta]
+   [metabase.mbql.util :as mbql.u])
+  #?(:cljs (:require [metabase.test-runner.assert-exprs.approximately-equal]))
+  #?(:cljs (:require-macros [com.gfredericks.test.chuck.clojure-test]
+                            [metabase.mbql.util])))
```

Comment:
This was necessary in old CLJS code, but it's no longer needed to require functions and namespaces separately from "consumer" namespaces.

Today all you have to do is set up the "provider" namespace properly. Either:
- Have separate `.clj` and `.cljs` files. `foo/bar/baz.cljs` should `(:require-macros [foo.bar.baz])` (and that might be the entire file).
- Have a single `.cljc` file that `(:require-macros [foo.bar.baz])` itself.

See `src/metabase/domain_entities/malli.clj{,s}` for an example of separate files, or [com.fredericks.test.chuck.clojure-test](https://github.com/gfredericks/test.chuck/blob/master/src/com/gfredericks/test/chuck/clojure_test.cljc) itself for a CLJC example.

If you fix `metabase.mbql.util` to `:require-macros` itself properly, both of these should just be `:require`d normally here.